src/pages/vouchers/redeem.tsx

```
@@ -0,0 +1,259 @@
+        return <Template3 {...templateProps} />;
+      case 'template4':
+        return <Template4 {...templateProps} />;
+      case 'template5':
+        return <Template5 {...templateProps} />;
+      default:
+        return <Template1 {...templateProps} />;
+    }
+  };
+
+  const isVoucherRedeemed = redeemed || (currentVoucher && (currentVoucher.isRedeemed || currentVoucher.status === 'redeemed'));
+  const isVoucherExpired = currentVoucher && currentVoucher.expirationDate 
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)